### ISSUE TITLE
ToDo

### ISSUE THREAD

**Issue by kir0ul** [DanNguyen1/DL-project#1]
Just listing some stuff I believe are still missing:

- [x] Metrics (i.e. accuracy, F1)
- [x] Prediction on train/~test~ set
- [x] Save/load model checkpoint
- [x] Train model on cluster on the whole dataset
- [x] Plots
  - [x] Training/testing accuracy vs. epochs
  - [x] Loss function
  - [x] Confusion matrix (if we have time)
  - [x] AUC ROC (if we have time)

**Comment by kir0ul** [comment]
- [x] Plot accuracy on both train & validation sets?
- [ ] ~~Accuracy trend line?~~
- [x] Add model checkpoint to the Gdrive
